scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -623,12 +635,6 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
   @inline final def commaSeparated[T <: Tree : AstMetadata](part: => T): List[T] =
     tokenSeparated[`,`, T](sepFirst = false, part)
```

Comment:
I wonder why there was a necessity to change `caseSeparated`, but there wasn't a need to change `commaSeparated`. Both seem to need support for ellipses.